intranet/apps/preferences/tests.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+from django.core.urlresolvers import reverse
+from ...test.ion_test import IonTestCase
+
+
+class PreferencesTest(IonTestCase):
+
+    settings_dict = {'showpictures-self': ['on'],
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Duplicate key 'showschedule-self' in dictionary](https://www.codacy.com/app/tjhsst-ion/ion/file/2380009964/issues/source?bid=3212528&fileBranchId=3221078#l8)